infra/ansible/config/pip.yaml

```diff
@@ -49,5 +49,6 @@ pip:
   # Packages that will be installed with the `--nodeps` flag.
   pkgs_nodeps:
     release_common:
+      - torch
```

Comment:
See this comment: https://github.com/pytorch/xla/pull/6439#pullrequestreview-1856977923

I agree with @vanbasten23's hack in the short term: https://github.com/pytorch/xla/pull/6439#issuecomment-1922481500

Basically, just add `pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu` to the TPU CI script. This is going to fail some of the time when the last nightly doesn't include required changes from the current head. But, failing some of the time is better than failing all of the time. TPU CI script: https://github.com/pytorch/xla/blob/master/test/tpu/xla_test_job.yaml

A short term hack is fine, since we're trying to replace the current TPU CI anyway. cc @mbzomowski

Reply:
Thanks, Will, make sense and TPUCI passed with this change: https://github.com/pytorch/xla/runs/21241385079

let's add this short term hack and wait for TPU CI replacement too